components/Styled/MealCard.js

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function MealCard({ recipe }) {
+  return (
+    <StyledLink href={`/recipe/${recipe._id}`}>
+      <StyledCard>
+        <StyledImage
+          src={recipe.imageLink}
+          alt={recipe.title}
+          height={123}
+          width={123}
+        />
+        <StyledDiv>
+          <StyledPTitle>{recipe.title}</StyledPTitle>
+          <StyledPDuration>
+            {recipe.duration} MIN | {recipe.difficulty.toUpperCase()}
+          </StyledPDuration>
+        </StyledDiv>
+      </StyledCard>
+    </StyledLink>
+  );
+}
+
+const StyledImage = styled(Image)`
+  border-radius: 20px 0px 0px 20px;
```

Comment:
if the value is 0, you can omit the units:

```suggestion
  border-radius: 20px 0 0 20px;
```